test-data/check-modules.yml

```diff
@@ -0,0 +1,17 @@
+# Type checker test cases dealing with modules and imports.
+
+- case: Access imported definitions
+  files:
```

Comment:
No, it's not possible to omit it here. In a sense, YAML is just a simpler way to write JSON objects, which are, from the Python's point of view, just dictionaries.

So, this YAML file:

```yaml
# Type checker test cases dealing with modules and imports.

- case: Access imported definitions
  files:
    - content: |
        ...
    - path: m.py
      content: |
        ...
```

is equivalent to the following Python dict:

```python
[{'case': 'Access imported definitions',
  'files': [{'content': '...\n'}, {'content': '...', 'path': 'm.py'}]}]
```

(You can convert a YAML string to a Python dict / list using `yaml.load(yaml_string)`.)

So we need the `files` attribute, as it becomes the `'files'` key in our test case dict.